layouts/robots.txt

```diff
@@ -0,0 +1 @@
+User-agent: *
```

Comment:
This overwrites https://github.com/cloudposse/docs/blob/master/static/robots.txt